Build/Build.cs

```diff
@@ -246,5 +258,22 @@ class Build : NukeBuild
                     (v, path) => v.SetTargetPath(path)));
         });
 
+    Target Spellcheck => _ => _
+        .Executes(() =>
+        {
+            InstallCSpell();
+            RunCSpell();
+        });
+
+    private void InstallCSpell()
+    {
+        Node($"{YarnCli} install --silent", workingDirectory: RootDirectory);
+    }
+
+    private void RunCSpell()
+    {
+        Node($"{YarnCli} run cspell");
```

Comment:
![image](https://user-images.githubusercontent.com/919634/210792882-a405be4d-c2b0-46a2-9529-48c3514fb33f.png)

https://github.com/fluentassertions/fluentassertions/actions/runs/3845958130/jobs/6550706354

Is there some exit code confusion between cSpell and NUKE 🤔

Reply:
Yes.. the reason for this is: cspell sends this to stderr. And NUKE takes this and prints an error.

I had this sort of "error" several times in my day job. (Was building docker images with NUKE).

I haven't found a solution yet

Reply:
Fixed :)